app/js/services/services.js

```diff
@@ -638,6 +640,34 @@ KMCServices.factory('PlayerService', ['$http', '$modal', '$log', '$q', 'apiServi
 				}
 				return playersService.playerVersionsMap;
 			},
+            'getPlayerProductVersion': function () {
+				if (playersService.playerProductVersion === undefined) {
+					var kmc = window.parent.kmc;
+					if (kmc && kmc.vars && kmc.vars.studioV3 && kmc.vars.studioV3.playerConfVars) {
+						try {
+							playersService.playerProductVersion = JSON.parse(kmc.vars.studioV3.playerConfVars);
+						} catch (e) {
+							console.error(e);
+						}
+					}
+					playersService.playerProductVersion = angular.isObject(playersService.playerProductVersion) &&  playersService.playerProductVersion.version ? playersService.playerProductVersion.version : "";
+				}
+				return playersService.playerProductVersion;
+			},
+            'getPlayerBetaProductVersion': function () {
```

Comment:
I would expose one api getPlayerProductVersion which gets param 'latest' to 'beta'

Reply:
done